README.md

```diff
@@ -1,2 +1,407 @@
+╭─ Options ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ --val-size                             FLOAT    validation size of text data to use for tuning [default: 0.8]                                                              │
+│ --nb-thresholds                        INTEGER  number of thresholds to be tried divided evenly between 0 and 1 [default: None]                                            │
+│ --init-threshold                       FLOAT    initial threshold value to compare against [default: 0.2]                                                                  │
+│ --split-data        --no-split-data             flag on whether to split data as was done for train [default: no-split-data]                                               │
+│ --help                                          Show this message and exit.                                                                                                │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+## ⬇️  Download
+
+This commands enables you to download mesh data from EPMC
+
+### epmc-mesh
+
+```
+
+ Usage: grants-tagger download epmc-mesh [OPTIONS] DOWNLOAD_PATH
+
+╭─ Arguments ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ *    download_path      TEXT  path to directory where to download EPMC data [default: None] [required]                                                                     │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Options ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ --year        INTEGER  year to download epmc publications [default: 2020]                                                                                                  │
+│ --help                 Show this message and exit.                                                                                                                         │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+# 🧑🏻‍💻  Develop
+
+Install development dependencies via:
+`poetry install --with dev`
+
+## 📖 Data
+
+If you work for Wellcome and have access to our AWS account,
+you easily download the raw data by typing `make sync_data`.
```

Comment:
this is not here anymore, we only use dvc from now on

Reply:
Removed this section entirely